### ISSUE TITLE
Readonly chips showing too much padding

### ISSUE THREAD

**Issue by laberle** [angular/material#5469]
Chip readonly mode is showing too much padding:

![screenshot from 2015-10-30 15 28 18](https://cloud.githubusercontent.com/assets/1323647/10858530/5ad8a884-7f1b-11e5-98d5-9aba38df4a2b.png)

Source code:

```
 <md-chips ng-model="ctrl.asdf" readonly="true">
  <md-chip-template>
     {{ $chip }}
  </md-chip-template>
</md-chips>
```

When I inspect the md-chip element that gets generated, its class is ```class="ng-scope md-chip md-default-theme md-readonly"```, but ```padding-right: 22px``` due the following CSS selector: 
```
.md-chips:not(.md-readonly) .md-chip {
    padding-right: 22px;
}
```

Chip HTML: 
```
<md-chip ng-repeat="$chip in $mdChipsCtrl.items" index="0"
  ng-class="{'md-focused': $mdChipsCtrl.selectedChip == $index, 'md-readonly': $mdChipsCtrl.readonly}" 
  class="ng-scope md-chip md-default-theme md-readonly">          
    <div class="md-chip-content" tabindex="-1" aria-hidden="true" 
        ng-focus="!$mdChipsCtrl.readonly && $mdChipsCtrl.selectChip($index)" 
        md-chip-transclude="$mdChipsCtrl.chipContentsTemplate">
      <md-chip-template class="small-chip-text ng-binding ng-scope">
        someText
      </md-chip-template>
    </div>          
    <!-- ngIf: !$mdChipsCtrl.readonly -->       
    <!-- ngIf: !$mdChipsCtrl.readonly -->
</md-chip>
```

The selector looks like it shouldn't be applied to this element, but it is (unapplying this selector fixes the problem): 

![image](https://cloud.githubusercontent.com/assets/1323647/10858556/8f3ac5d0-7f1b-11e5-859d-887e2e08ec41.png)

**Comment by JordanMajd** [comment]
Possibly duplicate of #4932

**Comment by topherfangio** [comment]
Fixed by #4621. Please comment if you still see issues and I will reopen.
